core/templates/components/button-directives/create-activity-button.component.spec.ts

```diff
@@ -146,8 +150,8 @@ describe('CreateActivityButtonComponent', () => {
       expect(component).toBeDefined();
     });
 
-  it('should begin activity creation process if user can' +
-    ' create collections', fakeAsync(() => {
+  it('should begin activity creation process if user can create collections' +
+    ' and is redirected to creator dashboard', fakeAsync(() => {
```

Comment:
Here's the issue I have with this, though. If you look at the body of this test, there is no mention of redirection to creator dashboard. So I think your test description is "breaking out of the boundaries" here. It should be self-contained with regards to the body of the test (i.e. the test body should obviously be setting up the conditions needed and then executing the test, and it should be easy to tie the test body back to the description).

Try again?

Reply:
Made some changes, PTAL